src/task2.py

```diff
@@ -351,4 +352,8 @@ def print_order(user_id, order_id):
         print("No item in order list!")
         return False
 
+    personal_bill = calc_cost(items, user_id, 4)
```

Comment:
Number of students should be dynamic instead of a hard coded "4". There is code in create_new_order from task1 that you can reuse.